tests/test_http_request.py

```diff
@@ -1642,6 +1642,26 @@ def test_replace_dumps_kwargs(self):
             self.assertEqual(kwargs["ensure_ascii"], True)
             self.assertEqual(kwargs["allow_nan"], True)
 
+    def test_replacement_both_body_and_data_warns(self):
+        """Test that we can get a warning if both body and data are passed for branch coverage"""
```

Comment:
Can you please remove the branch coverage mentions from here and other similar changes in other PRs? I don't think they are useful.